felix/policysync/processor.go

```diff
@@ -719,7 +719,7 @@ func splitIPSetDeltaUpdate(update *proto.IPSetDeltaUpdate) []proto.ToDataplane {
 			adds = adds[1:]
 		}
 
-		// Put removes on the message if they fit, but work from end end of the list since that is where
+		// Put removes only the message if they fit, but work from the end of the list since that is where
```

Comment:
Note that I don't' usually look at two-letter words. Both changes here were atypical.

Reply:
Here I think "on" was correct.

Reply:
Is this what it's trying to say?
```
		// Put "removes" on the message if they fit, but work from end end of the list since that is where
```
I had a really hard time trying to parse it. Is `Put` a _noun_ or a _verb_? Is `removes` a _noun_/_adjective_ or a _verb_?

Reply:
I'm going with `in` instead of `on` or `only`. The `message` is a container for things, so:

> `Put`ting _verb_ `removes` _noun (object)_ `in` _preposition_ `the message` _noun (object of preposition)_

Reply:
Yes, "Put" is a verb here, and "removes" is a plural noun.

Reply:
I'd like to stick with "on", because it is also correct usage, and I don't think it's fair to impose a change on the original author.

Reply:
Oh sorry, I see that it wasn't "on" before this PR, it was "only".  OK, happy to go with "in" then.

Reply:
D'oh, now I'm confusing myself.  It was "on" before, so I'll revert to that.